src/components/header/header.test.js

```diff
@@ -63,4 +65,16 @@ describe('Header', () => {
     wrapper = mountWithRouter(<Header {...mockInputProps} />, options);
     expect(wrapper.find('Countdown')).to.have.length(1);
   });
+
+  it('should display logout dialog window', () => {
```

Comment:
Do we need a test to assert that when the dialog is open, and the user confirms to renew timeout, the dialog is then closed?

Reply:
I'm creating a different task for rest of tests this one is already too big